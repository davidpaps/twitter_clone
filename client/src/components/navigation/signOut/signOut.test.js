import React from "react";
import { shallow } from "enzyme";
import SignOut from "./signOut";

describe("SignOut", () => {
  let wrapper;

  beforeEach(() => (wrapper = shallow(<SignOut />)));

  it("should render a <Link />", () => {
    expect(wrapper.find("Link").length).toEqual(1);
  });

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(1);
  });

  it("should render a <img />", () => {
    expect(wrapper.find("img").length).toEqual(1);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
