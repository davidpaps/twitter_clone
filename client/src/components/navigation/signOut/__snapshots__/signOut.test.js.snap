// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOut should render correctly 1`] = `ShallowWrapper {}`;
